release/tag/goreleaser.yaml

```diff
@@ -23,7 +23,7 @@ builds:
     goarch:
       - amd64
       - arm64
-    ldflags: -s -w -X github.com/GoogleContainerTools/kpt/run.version={{.Version}}
+    ldflags: -s -w -X github.com/{{ .Env._GITHUB_USER }}/kpt/run.version={{.Version}}
```

Comment:
I don't think we should change this. The package path would be the same regardless of whether it run on the `GoogleContainerTools` repo or somewhere else right?